shinydashboard/lantern/functions/endpoints.R

```diff
@@ -699,10 +717,10 @@ get_single_endpoint_locations <- function(db_connection, endpointURL, requestedF
           npi_id,
           match_score,
           left(zipcode,5) as zipcode
-        FROM endpoint_export where zipcode is NOT NULL AND match_score > .97 AND url = '", endpointURL, "' AND requested_fhir_version = '", requestedFhirVersion, "'"))
+        FROM organization_location where zipcode is NOT NULL AND match_score > .97 AND url = '", endpointURL, "' AND requested_fhir_version = '", requestedFhirVersion, "'"))
```

Comment:
I don't think we have to check the zipcode and match_code again here since we're already doing it in the organization_location view!